### PR TITLE
[cli] MAJOR: remove `vc deploy` clipboard copy feature

### PR DESCRIPTION
**Pull request by MatthewStanciu**
https://vercel.slack.com/archives/C03F2CMNGKG/p1656971502881949

Right now, `vc deploy` automatically copies the deploy url to your clipboard after the deployment has finished. You can opt out via the `--no-clipboard` flag, but the feature is enabled by default.

This is strange behavior—there's no indication that the CLI will hijack your clipboard, and you don't know it's been hijacked until after it happens.

This PR removes the clipboard copying feature as well as the `--no-clipboard` flag.

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by styfle — APPROVED** [review]
Looks good 👍  

I would take it a step further and remove the feature completely. Instead, rely on the terminal printing the necessary information and the user clicking to open the url.

**Comment by MatthewStanciu** [comment]
I'm on board with that!

File: packages/cli/src/commands/deploy/args.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const help = () => `
     -m, --meta                     Add metadata for the deployment (e.g.: ${chalk.dim(
       '`-m KEY=value`'
     )}). Can appear many times.
-    -C, --no-clipboard             Do not attempt to copy URL to clipboard
+    -C, --clipboard                Copy URL to clipboard after deployment
```
Comment (styfle) [inline packages/cli/src/commands/deploy/args.ts]: This should be removed

Also the docs need to be removed here https://vercel.com/docs/cli#introduction/unique-options/no-clipboard

**Review by styfle — CHANGES_REQUESTED** [review]
The help message needs to be updated.

Also the docs need to be removed here https://vercel.com/docs/cli#introduction/unique-options/no-clipboard

**Review by styfle — APPROVED** [review]

**Review by EndangeredMassa — APPROVED** [review]

File: test/benchmark.js [hunk 2]
```diff
@@ -3,9 +3,7 @@ const { tmpdir } = require('os');
 const { mkdirSync, writeFileSync } = require('fs');
```
Comment (TooTallNate) [inline test/benchmark.js]: I'm not even sure what this file is. Perhaps we should delete it if it's not being used by CI.

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**